tests/operations/tests/Mime.mjs

```diff
@@ -0,0 +1,22 @@
+/**
+ * Mime tests.
+ *
+ * @author bwhitn [brian.m.whitney@outlook.com]
+ * @copyright Crown Copyright 2018
+ * @license Apache-2.0
+ */
+import TestRegister from "../TestRegister";
+
+TestRegister.addTests([
```

Comment:
It would be nice to see some more tests here for both Decode Mime Encoded Words and Parse Mime